
# This message contains a list of all the current drone roles, formulated by the
# multi-agent strategy and read by each drone's IA.

# Role list
boris_drone/DroneRole[] roles
